src/Microsoft.CodeQuality.Analyzers/Core/ApiDesignGuidelines/ParameterNamesShouldMatchBaseDeclaration.cs

```diff
@@ -30,7 +30,7 @@ public sealed class ParameterNamesShouldMatchBaseDeclarationAnalyzer : Diagnosti
                                                                              DiagnosticHelpers.DefaultDiagnosticSeverity,
                                                                              isEnabledByDefault: false,
                                                                              description: s_localizableDescription,
-                                                                             helpLinkUri: null,     // TODO: add MSDN url
+                                                                             helpLinkUri: "https://msdn.microsoft.com/en-us/library/ms182251.aspx",
```

Comment:
❗️ https://docs.microsoft.com/en-us/visualstudio/code-quality/ca1725-parameter-names-should-match-base-declaration